BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -716,8 +720,6 @@ Robert Sparks and Dave Waltermire provided helpful reviews on behalf of the Gene
 
 During IESG review, Richard Barnes, Alissa Cooper, Spencer Dawkins, Stephen Farrell, Barry Leiba, Kathleen Moriarty, and Pete Resnick provided comments that led to further improvements.
```

Comment:
Please add a word about these acknowledgements applying to the previous version of the RFC.